examples/00-hello-world/graphql.php

```diff
@@ -9,7 +9,7 @@
 // Try mutation
 // curl -d '{"query": "mutation { sum(x: 2, y: 2) }" }' -H "Content-Type: application/json" http://localhost:8080
 
-require_once __DIR__ . '/../../vendor/autoload.php';
+require_once __DIR__ . '/vendor/autoload.php';
```

Comment:
but… this isn't correct?

Reply:
Sorry, I did not clone the repository, but took a separate file, I had a different structure. I am closing this PR.